test/watch.js

```diff
@@ -591,6 +597,8 @@ describe('Query watch', function() {
   };
 
   beforeEach(function() {
+    Backoff.setTimeoutHandler(setImmediate);
```

Comment:
So we are intentionally skipping the delays?  Can you add a comment?

Reply:
Added comment.